### PR TITLE
Allow discovery search results to be sorted

### PR DESCRIPTION
**Pull request by kluo**
@stvstnfrd

### CODE REVIEW

File: search/api.py [hunk 1]
```diff
@@ -103,6 +103,7 @@ def course_discovery_search(search_term=None, size=20, from_=0, field_dictionary
         filter_dictionary={"enrollment_end": DateRange(datetime.utcnow(), None)},
         exclude_dictionary=exclude_dictionary,
         facet_terms=course_discovery_facets(),
+        sort=getattr(settings, 'SEARCH_DISCOVERY_SORT_FIELDS', ''),
```
Comment (stvstnfrd) [inline search/api.py:106]: Is empty string the same as passing no string?

File: setup.py [hunk 2]
```diff
@@ -5,7 +5,7 @@
 
 setup(
     name='edx-search',
-    version='1.2.1',
+    version='1.2.2',
```
Comment (stvstnfrd) [inline setup.py]: Maybe this should be a minor version bump?
It's not really a bug fix; it's a backward-compatible feature improvement :)

**Review by stvstnfrd — CHANGES_REQUESTED** [review]

**Comment by stvstnfrd** [comment]
We should definitely send this upstream asap!

File: search/api.py (hunk 1 above)
Comment (kluo) [reply search/api.py:106]: Correct

**Review by stvstnfrd — APPROVED** [review]

**Comment by stvstnfrd** [comment]
Can you lob this over to edx?

Hopefully this will be an easy win 🤞

**Pull request merged**